src/EnergyPlus/SizingManager.cc

```diff
@@ -5361,4 +5364,91 @@ void UpdateTermUnitFinalZoneSizing(EnergyPlusData &state)
         }
     }
 }
+void checkZoneOutdoorAirFlowDesignBalanceForExhaustFan(EnergyPlusData &state)
+{
+    // Begin input and sizing check on the air balancing related to zone exhaust fan flows
+
+    static std::string const RoutineName("checkZoneOutdoorAirFlowDesignBalanceForExhaustFan");
+    // Step 1.  Run thru zones attached to an airloop and check if the zone minimum outdoor air flow is at least as large as the zone exhaust fan
+    // flows that are expected to be balanced by a central air system during operation.
```

Comment:
This seems overly restrictive. Often, but not always, kitchen or restroom exhaust is expected to be made up by mixing from neighboring zones.

Reply:
The zone exhaust fan flow should be no greater than the zone OA flow + infiltration + mixing. Right? During sizing you should have the zone OA flow + mixing, but what about infiltration? Should the exhaust fan flow be assumed to not include infiltration? I guess the minimum infiltration is 0 when there is no wind, so it sounds like the exhaust fan flow should be no grater than OA flow + mixing. That sounds like what @mjwitte said so there is at least some consensus on the test. Now mixing can have a schedule so there is a problem with the exhaust fan schedule vs the mixing schedule (unless mixing can be overridden to operate outside the mixing schedule). If the exhaust fan is ever on when mixing is off, then this method sounds like it should work. But if mixing is always active when the exhaust fan is scheduled to operate then the warning could be tripped inadvertently.